pkg/controller/issueapi/issue.go

```diff
@@ -310,7 +310,10 @@ func (c *Controller) HandleIssue() http.Handler {
 		}
 
 		if request.Phone != "" && smsProvider != nil {
-			if err := func() error {
+			// +1XXX55501XX is reserved for fictitious use. Just log for dev mode.
+			if c.config.DevMode() && request.Phone[:2] == "+1" && request.Phone[5:5] == "55501" {
```

Comment:
Is there a panic if I put in an invalid phone number

`+1123`

Reply:
I would move this to a helper / testable method

Reply:
good points. hows that?